Source/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyTracking</key>
+	<false/>
+</dict>
+</plist>
```

Comment:
This is how it looks in Xcode:
<img width="511" alt="image" src="https://github.com/ChartBoost/chartboost-mediation-ios-adapter-adcolony/assets/122156786/d46ef1bc-ca13-4001-88a7-7b326587dcc6">

Reply:
Should be fine, but just in case, are we sure that other keys are not necessary?
Just wondering if Xcode will consider the file invalid if fields like `NSPrivacyCollectedDataTypes` are not included, and if we should add them as empty lists.
Probably fine, just asking

Reply:
Checked WWDC video and the official doc https://developer.apple.com/documentation/bundleresources/privacy_manifest_files, but Apple didn't see whether these keys are required while there is no value. Most likely these are optional keys, but having the keys with empty value is probably safer and let me do that.